src/apps/chifra/pkg/names/custom.go

```diff
@@ -1,56 +1,222 @@
 package names
```

Comment:
I have a whole collection of custom names (for my own accounts and my friends) that is on my machine. I will make sure to see if they still work with zero changes to my configuration. You may resolve this, it's just for FYI.